process_report/process_report.py

```diff
@@ -164,6 +164,11 @@ def main():
         default="prepaid_contacts.csv",
         help="CSV listing all prepay group contact information. Defaults to 'prepaid_contacts.csv'",
     )
+    parser.add_argument(
+        "--coldfront-data-file",
+        required=False,
+        help="JSON file containing coldfront allocation data, must follow the format of the coldfront-plugin-api API",
+    )
```

Comment:
So, is this purely for test purposes? because if I were to call `main()` but don't set these:

```
REQUIRED_ENV_VARS = ("KEYCLOAK_CLIENT_ID", "KEYCLOAK_CLIENT_SECRET")
```

Then it will complain even though there's no reason to set those if I am providing a file. Obviously as a workaround I could just set these to whatever.

Reply:
Ah good point. I'll make the env vars optional if the `--coldfront-data-file` option is set